<ui version="4.0" >
 <class>CSwissRangerImageViewComp</class>
 <widget class="QWidget" name="CSwissRangerImageViewComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SwissRanger Image</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="DepthImage2dFrame" >
         <property name="title" >
          <string>Depth Image (2D)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" />
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="AmplitudeImageFrame" >
         <property name="title" >
          <string>Amplitude Image</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" />
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ConfidenceMapFrame" >
         <property name="title" >
          <string>Confidence Map</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" />
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="DepthImageFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Depth Image (3D)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" />
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
